breeze

```diff
@@ -16,146 +16,245 @@
+    # Runtime is empty initially (might be set to kubernetes in case kubernetes is chosen)
+    export RUNTIME=""
 
-#extra options for Docker compose
-EXTRA_DC_OPTIONS=()
+    # Do not enable Kind Kubernetes cluster by default
+    export ENABLE_KIND_CLUSTER="false"
 
-# If true, the docker images are rebuilt locally.
-export AIRFLOW_CONTAINER_DOCKER_BUILD_NEEDED="false"
+    # Do not recreate Kubernetes cluster by default
+    export RECREATE_KIND_CLUSTER="false"
 
-# By default we only pull images if we do not have them locally.
-# This can be overridden by -p flag
-export AIRFLOW_CONTAINER_FORCE_PULL_IMAGES="false"
+    # Do not stop Kubernetes cluster by default
+    export STOP_KIND_CLUSTER="false"
 
-# Start airflow-testing image with all the dependencies
-export AIRFLOW_CONTAINER_DEPS="true"
+    # We use docker image caches by default to speed up the builds
+    export USE_PULLED_IMAGES_AS_CACHE=${USE_PULLED_IMAGES_AS_CACHE:="true"}
 
-# Runtime is empty initially (might be set to kubernetes in case kubernetes is chosen)
-export RUNTIME=""
+    # By default we do not push images. This can be overridden by -u flag.
+    export PUSH_IMAGES=${PUSH_IMAGES:="false"}
 
-# Do not enable Kind Kubernetes cluster by default
-export ENABLE_KIND_CLUSTER="false"
+    # Determine version of the Airflow from version.py
+    AIRFLOW_VERSION=$(cat airflow/version.py - << EOF | python
+print(version.replace("+",""))
+EOF
+    )
+    export AIRFLOW_VERSION
 
-# Do not recreate Kubernetes cluster by default
-export RECREATE_KIND_CLUSTER="false"
+    # Verbosity in running ci scripts
+    export VERBOSE="false"
 
-# Do not stop Kubernetes cluster by default
-export STOP_KIND_CLUSTER="false"
+    # Whether to force build without checking if it is needed
+    export FORCE_BUILD_IMAGES=${FORCE_BUILD_IMAGES:="false"}
 
-# We use docker image caches by default to speed up the builds
-export AIRFLOW_CONTAINER_USE_PULLED_IMAGES_CACHE=${AIRFLOW_CONTAINER_USE_PULLED_IMAGES_CACHE:="true"}
+    # Files determining whether asciiart/cheatsheet are suppressed
+    SUPPRESS_CHEATSHEET_FILE="${MY_DIR}/.suppress_cheatsheet"
+    SUPPRESS_ASCIIART_FILE="${MY_DIR}/.suppress_asciiart"
 
-# By default we do not push images. This can be overridden by -u flag.
-export AIRFLOW_CONTAINER_PUSH_IMAGES=${AIRFLOW_CONTAINER_PUSH_IMAGES:="false"}
+    # Default values for flags
 
-# Skip building full CI image locally
-export AIRFLOW_CONTAINER_SKIP_CI_IMAGE="false"
+    _BREEZE_DEFAULT_BACKEND="sqlite"
+    _BREEZE_DEFAULT_KUBERNETES_MODE="git_mode"
+    _BREEZE_DEFAULT_KUBERNETES_VERSION="v1.15.3"
+}
+# End of initialisation here
 
-# Branch name of the base image used (usually master or v1-10-test or v1-10-stable)
-export AIRFLOW_CONTAINER_BRANCH_NAME=${AIRFLOW_CONTAINER_BRANCH_NAME:=${DEFAULT_BRANCH}}
+function initialize_virtualenv() {
+   # Check if we are in virtualenv
+   set +e
+   echo -e "import sys\nif not hasattr(sys,'base_prefix'):\n  sys.exit(1)" | "python${PYTHON_VERSION}"
```

Comment:
@potiuk did we ever determine what was causing the issues with pyenv virtualenvs?

Reply:
Ah yeah. Will take a close look at this. I tested it on Linux and made sone fixes to the initialize virtualenv but I will test it now with anaconda/pyenv/venv and any other *env I can put my hands on. I will do it in another PR though :)